library/math/num.lisp

```diff
@@ -214,7 +255,12 @@
   (define-num-wrapping U16 16)
   (define-num-wrapping U32 32)
   (define-num-wrapping U64 64)
-  (define-num-wrapping UFix #.+unsigned-fixnum-bits+))
+  (define-num-wrapping UFix #.+unsigned-fixnum-bits+)
+  
+  (define-unsigned-num-underflow U8 %unsigned-8-bit-underflow-handler 8)
```

Comment:
these are re-defining `Num` for types that already have it. Above, `define-num-wrapping` is already defining `-` and the like for each of the listed types.